icubam/analytics/plots/__init__.py

```diff
@@ -83,8 +69,9 @@ def plot_int(
   ls="solid",
   s=3,
   fill_below=False,
+  kind="quadratic"
 ):
-  f = scipy.interpolate.interp1d(x, y, kind="quadratic")
+  f = scipy.interpolate.interp1d(x, y, kind=kind)
```

Comment:
I haven't noticed it's quadratic interpolation. This can potentially overfit a lot.

Reply:
Yes, we can raise an error if it fails maybe?